### DISCUSSION TITLE
Consulta sobre el ejercicio 6 del TP 1

### DISCUSSION THREAD

**Discussion by IoachimusRoderici** [INGCOM-UNRN-P1/cursada-2023#25] [Preguntas y respuestas] [answered]
El ejercicio 6 del TP 1 dice:

> Escribir una función que, mediante restas sucesivas, obtenga el valor del cociente y el resto de dos números enteros.

``` c
int division_lenta(int dividendo, int divisor);

int cociente_lento(int dividendo, int divisor);
```

Pero los nombres de las funciones que da no coinciden con el enunciado; ¿no tendrían que ser así?:
``` c
int cociente_lento(int dividendo, int divisor);

int resto_lento(int dividendo, int divisor);
```
¿o entendí mal lo que hay que hacer?

**Comment by JeanFranco99** [comment, marked as answer]
creo que el nombre de la 2da función está mal. 
Porque de una división se obtiene el cociente 
La segunda debería llamarse resto_lento

**Comment by martinvilu** [comment]
> ¿o entendí mal lo que hay que hacer?

Nope, tu corrección es correcta
